test/002-user.test.js

```diff
@@ -5,25 +5,91 @@ import server from '../app';
 
 chai.use(chaiHttp);
 
-const { User } = models;
+const { User, Article } = models;
 const baseUrl = '/api/v1/users';
+const articleUrl = '/api/v1/articles';
 let resetToken;
 
 describe('USER TEST SUITE', () => {
-  let firstUserToken, firstUserID;
+  let firstUserToken, firstUserID, secondUserToken, thirdUserToken;
+  let article1Response, article2Response, article3Response;
 
   before(async () => {
     await models.sequelize.sync({ force: true });
 
-    const requestObject = {
+    const firstRequestObject = {
       username: 'johnsolomon',
       email: 'john@solomon.com',
       password: 'johnny777'
     };
 
-    const responseObject = await chai.request(server).post(`${baseUrl}`)
-      .send(requestObject);
-    firstUserToken = responseObject.body.token;
+    const secondRequestObject = {
+      username: 'davidsmith',
+      email: 'david@smith.com',
+      password: 'davsmith123'
+    };
+
+    const thirdRequestObject = {
+      username: 'janehook',
+      email: 'jane@hook.com',
+      password: 'jane4dhook'
+    };
+    const dummyArticles = [
```

Comment:
Could you extract all these request objects and arrays to a file called `testData` preferably inside the helpers directory? Then you can import any data you need and use as appropriate?

Reply:
I will do that now